test/GPU/spectral_transform.jl

```diff
@@ -1,3 +1,21 @@
+@testset "GPU spectral transform roundtrip " begin
+    spectral_grid = SpectralGrid(trunc=41, nlayers=8, architecture=GPU())
+    S = SpectralTransform(spectral_grid)
+    
+    # first roundtrip
+    L = randn(ComplexF32, spectral_grid.spectrum, 8)
+    field = zeros(Float32, spectral_grid.grid, 8)
+    transform!(field, L, S)
+    transform!(L, field, S)
+
+    # 2nd roundtrip
+    L2 = deepcopy(L)
+    transform!(field, L, S)
+    transform!(L, field, S)
+
+    @test L ≈ L2
```

Comment:
I would add here the 1-layer version too as only that will execute the serial FFT code otherwise you're only the the batched method!